binding/python/_orca.py

```diff
@@ -0,0 +1,396 @@
+        The pronunciation is expressed in ARPAbet format, e.g.: "I {live|L IH V} in {Sevilla|S EH V IY Y AH}".
+        :param output_path: Absolute path to the output audio file. The output file is saved as `WAV (.wav)`
+        and consists of a single mono channel.
+        :param speech_rate: Rate of speech of the generated audio.
+        :return: True if the operation is successful, throws an exception otherwise.
+        """
+
+        c_synthesize_params = self._get_c_synthesize_params(speech_rate=speech_rate)
+
+        status = self._synthesize_to_file_func(
+            self._handle,
+            text.encode("utf-8"),
+            c_synthesize_params,
+            output_path.encode("utf-8"))
+        if status is not self.PicovoiceStatuses.SUCCESS:
+            raise self._PICOVOICE_STATUS_TO_EXCEPTION[status](
+                message="Unable to synthesize speech",
+                message_stack=self._get_error_stack())
+
+        self._synthesize_params_delete_func(c_synthesize_params)
+
+        return True
+
+    @property
+    def version(self) -> str:
+        """Version."""
+
+        return self._version
+
+    def _get_c_synthesize_params(self, speech_rate: Optional[float] = None) -> POINTER(COrcaSynthesizeParams):
+        c_params = POINTER(self.COrcaSynthesizeParams)()
+
+        status = self._synthesize_params_init_func(byref(c_params))
+        if status is not self.PicovoiceStatuses.SUCCESS:
+            raise self._PICOVOICE_STATUS_TO_EXCEPTION[status](
+                message="Unable to create Orca synthesize params object",
+                message_stack=self._get_error_stack())
+
+        if speech_rate is not None:
+            status = self._synthesize_params_set_speech_rate_func(c_params, c_float(speech_rate))
+            if status is not self.PicovoiceStatuses.SUCCESS:
+                raise self._PICOVOICE_STATUS_TO_EXCEPTION[status](
+                    message="Unable to set Orca speech rate",
+                    message_stack=self._get_error_stack())
+
+        return c_params
+
+    def _get_error_stack(self) -> Sequence[str]:
+        message_stack_ref = POINTER(c_char_p)()
+        message_stack_depth = c_int()
+
+        status = self._get_error_stack_func(byref(message_stack_ref), byref(message_stack_depth))
+        if status is not self.PicovoiceStatuses.SUCCESS:
+            raise self._PICOVOICE_STATUS_TO_EXCEPTION[status](message="Unable to get Orca error state")
+
+        message_stack = list()
+        for i in range(message_stack_depth.value):
+            message_stack.append(message_stack_ref[i].decode("utf-8"))
+
+        self._free_error_stack_func(message_stack_ref)
+
+        return message_stack
+
+
+__all__ = [
+    'Orca',
+    'OrcaActivationError',
+    'OrcaActivationLimitError',
+    'OrcaActivationRefusedError',
+    'OrcaActivationThrottledError',
+    'OrcaError',
+    'OrcaInvalidArgumentError',
+    'OrcaIOError',
+    'OrcaMemoryError',
+    'OrcaRuntimeError',
```

Comment:
pretty sure they are more errors in other sdks, is there reason we exlucded them?

Reply:
Good catch, they were missing in the export but have been defined at the top of the file!